lib/chatBot.js

```diff
@@ -98,65 +120,56 @@ var ChatBot = function(username, password, options) {
 		json: false
 	});
 
-	if(!options.logFile || options.logFile !== false || options.logFile === null || options.logFile === undefined)  {
-		if(options.logFile === true) { that.logFile = "bot." + that.name + ".log"; }
-		else { that.logFile = options.logFile; }
-		that.winston.info("Chatbot "+that.name+" logging output to: " + that.logFile);
-		that.winston.add(
+	if(!fs.existsSync(this.name+"/firstRun")) {
+		var c = require('colors/safe');
+		console.error(c.white.bgBlack('=================='));
+		console.error(c.bgBlack.cyan("The location and names of default files has been changed."));
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/968187183/issues/source?bid=1980036&fileBranchId=2848073#l127)